### DISCUSSION TITLE
How to improve Performance with 298k products?

### DISCUSSION THREAD

**Discussion by merdiano** [bagisto/bagisto#6415] [Q&A]
Getting products is slow and take 24 seconds with 298k products.
Tested on server with 4 cpu and 6gb ram
mysql 8.0.29
<img width="1817" alt="sargaslow" src="https://user-images.githubusercontent.com/8520650/167408411-43344177-f8d0-4d47-ba70-25539b2c4b96.png">

How can i decrease this query execution time on getting products? Any suggestions?

**Comment by Gami-404** [comment]
Hi , 
There i love the performance challenging ,  can help you we can get meeting on the zoom , that totally for FREE

**Reply by Lwabyaeric** [reply]
No problem Whatsapp me +256706560262

**Comment by Plansol** [comment]
Hi @merdiano, i have similar issue, did you solve this problem?

thanks

**Comment by ali-apadana** [comment]
hi, try elasticsearch

**Comment by devansh-webkul** [comment]
Latest version is having elasticsearch support and many big processes are already in background. Try with latest version and for more information adn engagement you can checkout the Bagisto forum.
